lib/assets/javascripts/cable.js.coffee

```diff
@@ -5,4 +5,8 @@
   PING_IDENTIFIER: "_ping"
 
   createConsumer: (url) ->
+    token = $("meta[name=csrf-token]").attr("content")
```

Comment:
@lsylvester  Yep that is a valid case. In my test with http://github.com/rails/actioncable-examples I had to move  the `csrf_meta_tags` before `javascript_include_tag`. I think we should move the cable connection creating code in a domready event. We can do that without jquery as browsers which support websockets support also DOMContentLoaded (see [this](http://caniuse.com/#feat=websockets), [this](http://caniuse.com/#feat=domcontentloaded) and [this](http://beeker.io/jquery-document-ready-equivalent-vanilla-javascript)).
@javan will do that